### PR TITLE
Port to Windows

### LINKED ISSUE #147: Write Windows port

### PR DESCRIPTION
**Pull request by jacobmarble**
This change adds support for Windows. The connector and util (renamed gcp-connector-util) build and run properly, but bugs are certain to appear. Some lingering issues:

- there is no logging by default, but --log-to-console works
- the connector needs to be adapted to be a service
- the connector fails to run outside of the MSYS2/MinGW64 environment

Fixes #147

### CODE REVIEW

File: winspool/win32.go [hunk 1]
```diff
@@ -0,0 +1,1465 @@
...
+	if err != nil {
+		return nil, err
+	}
+	var printerInfo8 PrinterInfo89 = *(*PrinterInfo89)(unsafe.Pointer(&pPrinter[0]))
+	var devMode *DevMode = (*DevMode)(unsafe.Pointer(printerInfo8.pDevMode))
+
+	return devMode, nil
+}
+
+func (hPrinter HANDLE) EnumPrinterDataEx() (map[string]interface{}, error) {
```
Comment (bvsbvs) [inline winspool/win32.go]: I don't see where EnumPrinterDataEx is used.

Comment (jacobmarble) [inline winspool/win32.go]: There are several such functions in this file. They are Windows API wrappers, so I chose to leave them for future use.

Comment (bvsbvs) [inline winspool/win32.go]: I'd recommend to remove this code, and keep that maybe in another branch.
PR :+1

**Comment by bvsbvs** [comment]
lgtm

File: winspool/win32.go (hunk 1 above)
Comment (jacobmarble) [inline winspool/win32.go]: Done.

**Comment by jacobmarble** [comment]
@bvsbvs PTAL and merge

**Pull request merged**